File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.py
def solution(s):
    # 문자열을 배열로 바꾸기
    array_s = list(s)
    # 배열을 정렬하기
    array_s.sort(reverse=True)
    # 다시 문자열로 바꾸기
    return "".join(array_s)

File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.java
import java.util.*;
class Solution {
    public String solution(String s) {
        /*
        큰 것부터 작은 것 순
        문자열 반대 정렬
        */
        
        char[] chars = s.toCharArray();
        Arrays.sort(chars);
        StringBuilder sb = new StringBuilder(new String(chars));
        String result = sb.reverse().toString();
        return result;
    }
}